src/main/app/claims/models/claim.ts

```diff
@@ -302,7 +307,9 @@ export class Claim {
       return true
     }
 
-    return (((this.response && (this.response as FullAdmissionResponse).paymentIntention.paymentOption !== PaymentOption.IMMEDIATELY && !this.isSettlementReachedThroughAdmission()
+    return (((this.response && (this.response as FullAdmissionResponse).paymentIntention &&
```

Comment:
just feels messy :))) not sure what to suggest though? maybe breaking out into const booleans or refactor into helper methods somewhere? its fine though...so maybe we do as refactor task another day

Reply:
Helper method would be preferred here.